jest.config.js

```diff
@@ -2,16 +2,16 @@ module.exports = {
   collectCoverage: true,
   // Ensures that we collect coverage from all source files, not just tested
   // ones.
-  collectCoverageFrom: ['./src/**.ts'],
+  collectCoverageFrom: ['**/src/**/*.ts'],
```

Comment:
Nit: We only have one `src` directory, so this would suffice:
```suggestion
  collectCoverageFrom: ['*/src/**/*.ts'],
```

Reply:
That doesn't work according to: https://globster.xyz/ 🤔

Reply:
Sorry, typo, I meant this:
```suggestion
  collectCoverageFrom: ['./src/**/*.ts'],
```

The added asterisk is the fix here I think

Reply:
Weirdly that doesn't seem to be enough 🤔

Reply:
Oh, nevermind that must be an issue with globster.